src/ray/core_worker/core_worker.h

```diff
@@ -256,6 +259,18 @@ class CoreWorkerProcess {
 
   /// To protect accessing the `workers_` map.
   mutable absl::Mutex worker_map_mutex_;
+
+  /// Event loop where the metric exporting are handled. e.g. async gRPC operations.
+  static boost::asio::io_service stats_io_service_;
+
+  /// Stats io worker keep stats io service alive.
+  static boost::asio::io_service::work stats_io_work_;
```

Comment:
I think `stats_io_service_` and `stats_io_work_` could be local variables in the callback of `stats_thread_`. No need to define them here.

Reply:
@kfstorm There is no such api for stopping opencensus stats and its threads, which is why we actually can not shutdown them down even if the thread is joinable.